### PR TITLE
Addresses a bug where Kubecost would report cost metrics in the billions

### PR DESCRIPTION
**Pull request by bstuder99**
## Related issue #

See https://github.com/kubecost/features-bugs/issues/60

### CODE REVIEW

**Comment by srpomeroy** [comment]
Do we want to be enabling custom prices even for environment that are cloud-based where we support reconciliation?

**Review by michaelmdresser — DISMISSED** [review]

**Comment by michaelmdresser** [comment]
Oh, true, good point from spom -- answer: probably not

File: architecture/pricing-sources-matrix.md [hunk 1]
```diff
@@ -60,6 +60,8 @@ kubecostProductConfigs:
     internetNetworkEgress: 0.12 # per GB per month cost
 ```
 
+Setting `customPricesEnabled` to `true` or enabling custom pricing through the UI has been shown to fix an error where Kubecost reported absurdly high cost metrics.
```
Comment (AjayTripathy) [inline architecture/pricing-sources-matrix.md:63]: I don't think this is right; I think this should just be a line about rancher kubernetes engine clusters needing to have custom prices enabled.

Comment (bstuder99) [reply architecture/pricing-sources-matrix.md:63]: If this only affects Rancher setups, we should just move this content into this article: https://docs.kubecost.com/install-and-configure/install/provider-installations/istio-rancher

Comment (AjayTripathy) [reply architecture/pricing-sources-matrix.md:63]: afaik it's only rancher.

**Comment by bstuder99** [comment]
I've removed the initial changes and added the content into a new Troubleshooting section for our Istio/Rancher install guide

**Pull request merged**